### PR TITLE
Issue #410: Install the redis PHP extension in Tugboat

### LINKED ISSUE #410: Install the PHP Redis extension when including Redis in Tugboat
We currently detect if redis is used in ddev and add redis as a service in tugboat. However, we don't actually install the PHP Redis extension, so users need to manually add that. Let's add something like:

```
        pecl -q install redis-5.3.7 # Pin a version? Good question!
        docker-php-ext-enable opcache redis
```

to our Tugboat init steps.

### PR DESCRIPTION
**Pull request by justafish**
Fixes #410

### CODE REVIEW

File: scaffold/tugboat/steps/1-init.sh.twig [hunk 1]
```diff
@@ -49,12 +49,16 @@ docker-php-ext-install gd
 # of GD. Lullabot will likely be publishing an ADR recommending it too.
 apt-get install -y imagemagick
 {% if memory_cache_type == "memcached" %}
-# Install the PHP memcache extension. The Drupal Redis module recommends
-# phpredis so no PHP extension is required.
+# Install the PHP memcache extension.
 
 apt-get install -y zlib1g-dev
-pecl install memcache
+yes '' | pecl install -f memcache
```
Comment (deviantintegral) [inline scaffold/tugboat/steps/1-init.sh.twig:55]: Interesting, I've never needed to do any of this in tugboat or CI - just `pecl install <something>`. What's up here?

File: scaffold/tugboat/steps/1-init.sh.twig [hunk 2]
```diff
@@ -49,12 +49,16 @@ docker-php-ext-install gd
...
 echo 'extension=memcache.so' > /usr/local/etc/php/conf.d/memcache.ini
+{% elseif memory_cache_type == "memcached" %}
```
Comment (deviantintegral) [inline scaffold/tugboat/steps/1-init.sh.twig:57]:
```suggestion
{% elseif memory_cache_type == "redis" %}
```

**Comment by deviantintegral** [comment]
Test failure is maximum multidev limit: https://github.com/Lullabot/drainpipe/issues/408

File: scaffold/tugboat/steps/1-init.sh.twig (hunk 1 above)
Comment (justafish) [reply scaffold/tugboat/steps/1-init.sh.twig:55]: it should run fine in non-interactive mode, but won't run without asking questions if you put that directly into the tugboat terminal - bit hard for me to test at the moment with our setup so just left it at what I can test directly

**Review by deviantintegral — COMMENTED** [review]
Looks good to me!

I actually expected this to fail tests though, given there's no corresponding test update.

**Review by deviantintegral — APPROVED** [review]

**Pull request merged**